src/python/sortingview/gui/extensions/unitstable/Units/Units.tsx

```diff
@@ -82,7 +83,7 @@ const Units: React.FunctionComponent<SortingViewProps & OwnProps> = (props) => {
         const name = metric.name;
 
         if (name in metrics) {
-            return metrics[name];
+            return
```

Comment:
? Not returning anything any more?

Reply:
It was never supposed to return anything. Don't know why it was.